templates/focus-ios.xliff

```diff
@@ -729,6 +749,74 @@
+        <source>Lock %@ when a site is open:</source>
+        <note>Text for a label that indicates the title for biometric tip.</note>
+      </trans-unit>
+      <trans-unit id="Tip.BiometricFaceId.Description">
+        <source>Turn on Face ID</source>
+        <note>Text for a label that indicates the description for biometric Face ID tip.</note>
+      </trans-unit>
+      <trans-unit id="Tip.BiometricTouchId.Description">
+        <source>Turn on Touch ID</source>
+        <note>Text for a label that indicates the description for biometric Touch ID tip.</note>
+      </trans-unit>
+      <trans-unit id="Tip.Release.Description">
+        <source>Read more about this and other updates to %@.</source>
+        <note>Text for a label that indicates the description for release tip.</note>
+      </trans-unit>
+      <trans-unit id="Tip.Release.Title">
+        <source>Why yes, we do have a fresh new look!</source>
+        <note>Text for a label that indicates the title for release tip.</note>
+      </trans-unit>
+      <trans-unit id="Tip.RequestDesktop.Description">
+        <source>Page Actions &gt; Request Desktop Site</source>
+        <note>Text for a label that indicates the description for request desktop tip.</note>
+      </trans-unit>
+      <trans-unit id="Tip.RequestDesktop.Title">
+        <source>Want to see the full desktop version of a site?</source>
+        <note>Text for a label that indicates the title for request desktop tip.</note>
+      </trans-unit>
+      <trans-unit id="Tip.ShareTrackers.Description">
+        <source>%@ trackers blocked so far</source>
+        <note>Text for a label that indicates the description for share trackers tip.</note>
+      </trans-unit>
+      <trans-unit id="Tip.ShareTrackers.Title">
+        <source>You browse. %@ blocks.</source>
+        <note>Text for a label that indicates the title for share trackers tip.</note>
+      </trans-unit>
+      <trans-unit id="Tip.Shortcuts.Description">
+        <source>Select Add to Shortcuts from the %@ menu</source>
+        <note>Text for a label that indicates the description for shortcuts tip.</note>
+      </trans-unit>
+      <trans-unit id="Tip.Shortcuts.Title">
+        <source>Create shortcuts to the sites you visit most:</source>
+        <note>Text for a label that indicates the title for shortcuts tip.</note>
+      </trans-unit>
+      <trans-unit id="Tip.SiriErase.Description">
+        <source>Add Siri shortcut</source>
+        <note>Text for a label that indicates the description for siri erase tip.</note>
+      </trans-unit>
+      <trans-unit id="Tip.SiriErase.Title">
+        <source>“Siri, erase my %@ session.”</source>
+        <note>Text for a label that indicates the title for siri erase tip.</note>
+      </trans-unit>
+      <trans-unit id="Tip.SiriFavorite.Description">
+        <source>Add Siri shortcut</source>
+        <note>Text for a label that indicates the description for siri favorite tip.</note>
```

Comment:
Comment also confusing here because doesn't explain how that shortcut is related to a favorite tip